chore(tsc): add Maya as TSC member

Mayaleeeee opened the pull request:
I have added my details to the yaml file.
Thank you.

derberg commented on MAINTAINERS.yaml:
```diff
@@ -1,4 +1,12 @@
----
+- name: Aishat Muibudeen
+  github: mayaleeeee
+  linkedin: aishatmuibudeen
+  slack:    U03CNHGEUR1
+  twitter:  maya_ux_ui
+  availableForHire: true
+  isTscMember: true
```
```suggestion
- name: Aishat Muibudeen
  github: mayaleeeee
  linkedin: aishatmuibudeen
  slack: U03CNHGEUR1
  twitter: maya_ux_ui
  availableForHire: true
  isTscMember: true
```

derberg reviewed (commented):
Welcome!
Please commit one suggestion that I added, it is related to empty spaces.

also for future remember that PR titles need to follow https://www.conventionalcommits.org/en/v1.0.0/. For now I fixed the PR title for you, please look at the change so you are aware the change direction

Mayaleeeee commented:
> Welcome! Please commit one suggestion that I added, it is related to empty spaces.
> 
> also for future remember that PR titles need to follow https://www.conventionalcommits.org/en/v1.0.0/. For now I fixed the PR title for you, please look at the change so you are aware the change direction

Thank you for the clarification and for fixing the PR title. I have implemented the necessary changes and will take note of the conventional commit guidelines in the future. Please let me know if there is anything else you want me to do.

derberg reviewed (approved):
all good, welcome!!!

derberg commented:
/rtm

derberg commented:
@Mayaleeeee  Soon you will be invited to join the AsyncAPI GitHub organization and you will be added to the team that lists all TSC members.

We use this team to mention in different discussions in different places in GitHub where TSC opinion is needed or even voting on some topic. Once TSC is mentioned:

- You get GitHub notification
- We also drop notification in our slack in #95_bot-tsc-members-mentioned channel
- We drop an email to people that subscribed to TSC news here https://www.asyncapi.com/community/tsc

Pick the channel for notifications that you prefer. Welcome aboard! We are excited to have you as part of the team.

Mayaleeeee commented:
> @Mayaleeeee  Soon you will be invited to join the AsyncAPI GitHub organization and you will be added to the team that lists all TSC members.
> 
> We use this team to mention in different discussions in different places in GitHub where TSC opinion is needed or even voting on some topic. Once TSC is mentioned:
> 
> - You get GitHub notification
> - We also drop notification in our slack in #95_bot-tsc-members-mentioned channel
> - We drop an email to people that subscribed to TSC news here https://www.asyncapi.com/community/tsc
> 
> Pick the channel for notifications that you prefer. Welcome aboard! We are excited to have you as part of the team.

GitHub notification is cool.
And I'm happy to be a member the great TSC🙏🙏.

Thank you once again.
